docs/quickstart.md

```diff
@@ -38,14 +38,20 @@ All further instructions assumes you are in the code directory eg. `ocp4-upi-pow
 
 Update the [var.tfvars](../var.tfvars) based on your environment. Description of the variables are available in the following [link](./var.tfvars-doc.md).
 
+For sensitive variables that should not be saved on disk, set these environment variables;
```

Comment:
How about rephrasing it to - 
"You can use environment variables for sensitive variables that should not be saved to disk"

Reply:
Sure!